packages/functions/src/jobs/CronJobs.ts

```diff
@@ -0,0 +1,69 @@
+import { AppointmentStatus } from "common/src/types";
+import * as functions from "firebase-functions";
+import { getAppointmentsByStatus } from "../dal/AppointmentDataAccessLayer";
+import { getDonors } from "../dal/DonorDataAccessLayer";
+import { getAppointmentNotificationData } from "../notifications/AppointmentNotificationData";
+import {
+  NotificationToDonor,
+  sendEmailToDonor,
+} from "../notifications/NotificationSender";
+
+export const ConfirmationReminderOnSameDay = functions.pubsub
+  .schedule("every 1 hours")
+  .onRun(async () => {
+    const topOfThisHourInMillis = new Date().setMinutes(0, 0, 0);
+    const oneHourInMillis = 1000 * 60 * 60;
+    // Offset by 1 ms, to avoid catching XX:00 twice
+    const topOfPreviousHourInMillis =
+      topOfThisHourInMillis - oneHourInMillis + 1;
+
+    SendConfirmationReminders(
+      new Date(topOfPreviousHourInMillis),
+      new Date(topOfThisHourInMillis)
+    );
+  });
+
+// Run every day at 11 AM Israel Time
+//https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
+export const ConfirmationReminderOnNextDay = functions.pubsub
+  .schedule("* 11 * * *")
+  .timeZone("Asia/Jerusalem")
+  .onRun(() => {
+    const midnightOfToday = new Date().setHours(0, 0, 0, 0);
+    const oneDayInMillis = 1000 * 60 * 60 * 24;
+    const midnightOfYesterday = midnightOfToday - oneDayInMillis;
```

Comment:
No need for 1ms buffer here as well?

Reply:
Are we expecting donations at midnight?

Reply:
It could be, we should be prepared for it.

Also, I am not sure line 32 will result in midnight, because this depends on the timezone the job is running at.

I didn't comment on this before because I am ok with it running daily at 02:00 or something like it